aea/helpers/git.py

```diff
@@ -32,6 +32,10 @@ def check_working_tree_is_dirty() -> None:
     if len(result) > 0:
         print("Git working tree is dirty:")
         print(result.decode("utf-8"))
+        print(
+            "You should bump PROTOCOL_GENERATOR_VERSION in aea/protocols/__init__.py "
```

Comment:
This method is generic no, whereas this message is specific to protocol generator check...